subprojects/core/src/main/java/org/gradle/internal/progress/DefaultBuildOperationExecutor.java

```diff
@@ -309,7 +309,6 @@ public String getDisplayName() {
         @Override
         public void execute(CallableBuildOperation<T> buildOperation, BuildOperationContext context) {
             returnValue = buildOperation.call(context);
-            System.out.print("");
```

Comment:
I have confirmed it was removed on `master`. Since this branch has no conflict with `master` now, I'd like to keep this change in case troublesome conflict occurs.